shiny/workflowPlot/ui.R

```diff
@@ -14,10 +14,30 @@ ui <- shinyUI(fluidPage(
       selectizeInput("all_run_id", "Mutliple Run IDs", c(),multiple=TRUE),
       actionButton("load", "Load Model outputs"),
       selectInput("variable_name", "Variable Name", ""),
-      radioButtons("plotType", "Plot Type", c("Scatter Plot" = "scatterPlot", "Line Chart" = "lineChart"), selected="scatterPlot")
+      radioButtons("plotType", "Plot Type (for Model Outputs)", 
+                   c("Scatter Plot" = "scatterPlot", 
+                     "Line Chart" = "lineChart"), 
+                   selected="scatterPlot"),
+      # uiOutput("slider"),
+      sliderInput("smooth_n", "Value for smoothing:",
+                  min=0, max=100, value=80),
+      tags$hr(),
+      tags$hr(),
+      fileInput('fileUploaded', 'Choose file to upload data',
+                accept=c('text/csv', 
+                         'text/comma-separated-values,text/plain', 
+                         '.csv')),
```

Comment:
I don't see a reason to limit the file upload type here. If you did want to restrict the type, it should be based on the FORMAT record specifying the allowable mime-type for that specific format record. Even if you don't restrict the upload that way, it seems like a pretty essential check to perform before using the data.